### ISSUE TITLE
kubegems-gitea

### ISSUE THREAD

**Issue by aaa123456-ch** [kubegems/kubegems#125]
版本v1.21.0

[root@k8s-master01 ~]# kubectl describe -n kubegems pod/kubegems-gitea-0
......
.......
Events:
  Type     Reason            Age                  From               Message
  ----     ------            ----                 ----               -------
 Warning  FailedScheduling  2m13s                default-scheduler  0/3 nodes are available: 3 pod has unbound immediate PersistentVolumeClaims.
  Warning  FailedScheduling  56s (x1 over 2m11s)  default-scheduler  0/3 nodes are available: 3 pod has unbound immediate** PersistentVolumeClaims.

[root@k8s-master01 ~]# kubectl edit statefulset.apps/kubegems-gitea -n kubegems
.....
.....
  volumeClaimTemplates:
  - apiVersion: v1
    kind: PersistentVolumeClaim
    metadata:
      creationTimestamp: null
      name: data
    spec:
      accessModes:
      - ReadWriteOnce
      resources:
        requests:
          storage: 10Gi
      volumeMode: Filesystem
    status:
      phase: Pending

这里好像没有storageClassName配置，是我漏了那一步吗？
部署：https://www.kubegems.io/docs/installation/quick-install

**Comment by cnfatal** [comment]
There is a wrong usage:

![image](https://user-images.githubusercontent.com/15731850/175472796-a3430ccb-ef82-465c-b001-f824c70ecbb2.png)
![image](https://user-images.githubusercontent.com/15731850/175472838-411abd3a-d70c-4bc2-8aa9-6734dc5d5778.png)

It should be:

```yaml
        {{ include "common.storage.class" ( dict "persistence" .Values.persistence "global" .Values.global) }}
```

Already fixed and repost gitea chart as `5.0.8` to kubegems chartmusem.

**Comment by cnfatal** [comment]
> 这里好像没有storageClassName配置，是我漏了那一步吗？

是bug，作为暂时性的修复。你可以将你的一个storageclass设置为默认，这样未指定storageclass 的PVC会自动绑定到默认storage class 。 参见：[改变默认 StorageClass](https://kubernetes.io/zh-cn/docs/tasks/administer-cluster/change-default-storage-class/)

稍后会放出出修复的版本
